modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -207,9 +207,7 @@ definition {
 
 		ObjectAdmin.deleteActionViaUI(actionName = "Action Name");
 
-		ObjectAdmin.assertObjectActionsNotPresent(
-			actionActive = "Yes",
-			actionName = "Action Name");
+		ObjectAdmin.assertObjectActionsNotPresent(actionName = "Action Name");
```

Comment:
The path has been modified and is only used to see the name, it does not work anymore to see if it is active.